src/hooks/usePoolData.ts

```diff
@@ -157,7 +168,10 @@ export default function usePoolData(name?: string): PoolDataHookReturnType {
           ...emptyPoolData,
           name: poolName || "",
         })
-        setUserShare(null)
+        setUserShare((prev) => ({
+          ...prev,
+          name: poolName || "",
```

Comment:
why set `poolName` here if this block is for when `poolName == null`? Why is it necessary to explicitly set name here?

Reply:
You are correctly, I followed only the poolData logic but we don't need to give poolName here because we initialized poolName with "".